### ISSUE TITLE
Suggestion: Once a File is Fixed Have Microsoft Word Open It

### ISSUE THREAD

**Issue by socrtwo** [desjarlais/wordcorruptdocchecker#6]
If Word Corrupt Doc Checker is successful in repairing a bad tag file, programatically have the software open Microsoft Word or its default DOCX file handler and load the fixed file.

**Comment by desjarlais** [comment]
I'll start playing around with this idea and maybe add an optional button to launch the file after the fix has been attempted.

**Comment by socrtwo** [comment]
That sounds like a good option.

Best Wishes,

Paul D Pruitt

socrtwo@s2services.com
(301) 493-4982
9006 Friars Rd.
Bethesda, MD  20817-3320

   - Have a manuscript lying around gathering dust? Let me help you
   self-publish it <socrtwo@s2services.com>.

**Comment by desjarlais** [comment]
initial work here has been added, it will be pushed into the changes soon.

**Comment by socrtwo** [comment]
Have you pushed this into the changes yet?

Best Wishes,

Paul D Pruitt

socrtwo@s2services.com
(301) 493-4982
9006 Friars Rd.
Bethesda, MD  20817-3320

   - Have a manuscript lying around gathering dust? Let me help you
   self-publish it <socrtwo@s2services.com>.

**Comment by desjarlais** [comment]
Not yet, still doing some testing, but it should be in soon.

**Comment by desjarlais** [comment]
work is still ongoing with this one.

**Comment by desjarlais** [comment]
I'm still working through the scenarios that can happen for this type of behavior.  It is taking longer than I initially expected, sorry for the delay.

**Comment by desjarlais** [comment]
Added a very generic simple way of doing this (it is in the Settings dialog I'm adding) and should be available in 1.2.1.8 when I add that release sometime this week.  I'm not really doing any error handling and I haven't tested any different scenarios so I imagine this could be a buggy feature.  I'll close this, but if you come across any problems with it, feel free to submit new bugs and I'll do what I can.

**Comment by socrtwo** [comment]
Cool, thanks.

Best Wishes,

Paul D Pruitt

socrtwo@s2services.com
(301) 493-4982
9006 Friars Rd.
Bethesda, MD  20817-3320

   - Have a manuscript lying around gathering dust? Let me help you
   self-publish it <socrtwo@s2services.com>.

**Comment by socrtwo** [comment]
I tried the new version and the open repaired file with Word setting. I
tried it on a customer's corrupt Word file and it worked opening with Word.
I didn't try the remove all fallbacks setting, because the file got fixed
without checking it.

I recommended your app to the user if the issue came up again.

Best Wishes,

Paul D Pruitt

socrtwo@s2services.com
(301) 493-4982
9006 Friars Rd.
Bethesda, MD  20817-3320

   - Have a manuscript lying around gathering dust? Let me help you
   self-publish it <socrtwo@s2services.com>.
